export_report.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python3
+
+import plotly.io as pio
+import pandas as pd
+from docx import Document
+from docx.shared import Inches
+import os
+import re
+import sys
+import datetime
+
+def generateGraphs():
+    graphs = {}
+    for root, _, files in os.walk("."):
```

Comment:
This could probably use a comment to describe what this section is doing and why